presentation/src/main/java/com/architect/coders/mu8/splash/SplashActivity.kt

```diff
@@ -0,0 +1,36 @@
+package com.architect.coders.mu8.splash
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.architect.coders.mu8.MainActivity
+import com.architect.coders.mu8.R
+import com.architect.coders.mu8.splash.SplashViewModel.UiModel
+import com.architect.coders.mu8.utils.startActivity
+
+class SplashActivity : AppCompatActivity() {
+
+    private lateinit var viewModel: SplashViewModel
+    private lateinit var tvVersion: TextView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+        tvVersion = findViewById(R.id.tv_version)
```

Comment:
Add a new line above this to improve reading.

Reply:
ok, new line

Reply:
@jamacado the line was after the `setContentView`